### PR TITLE
Use Webpack to get version

### PR DESCRIPTION
**Pull request by piwysocki**
- version from package.json set using webpack and used in cli.ts
- file.ts - loadFileSync - removed second parameter as it is called always with "yaml" parameter

### CODE REVIEW

**Comment by adrianmroz-allegro** [comment]
PR description doesn't explain changes to file.ts

File: src/server/cli/load-config-file.ts [hunk 1]
```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 import { Command } from "commander";
-import { loadFileSync } from "../utils/file/file";
+import { loadJsonFileSync } from "../utils/file/file";
 
 export function loadConfigFile(configPath: string, program: Command): object {
   try {
-    return loadFileSync(configPath, "yaml");
+    return loadJsonFileSync(configPath);
```
Comment (adrianmroz-allegro) [inline src/server/cli/load-config-file.ts:21]: > [in] loadFileSync [..] removed second parameter as it is called always with "json" parameter

I mean there is a call with "yaml" flag. And configs are written in yaml always ...

Comment (piwysocki) [reply src/server/cli/load-config-file.ts:21]: sorry, I'm blind

File: src/server/cli.ts [hunk 2]
```diff
@@ -30,19 +30,13 @@ import {
...
 import { parseCredentials } from "./cli/utils";
-import { readVersion } from "./version";
 
-let version: string;
-try {
-  version = readVersion();
-} catch (e) {
-  program.error(`Failed to read turnilo version. Error: ${e.message}`);
-}
+declare const VERSION: string;
```
Comment (plesiecki) [inline src/server/cli.ts:34]: what do you think about adding a comment about where is it actually defined?

Comment (piwysocki) [reply src/server/cli.ts]: changed variable name to be self explanatory

**Review by plesiecki — APPROVED** [review]

**Pull request closed without merging**